CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
-## Fleet 4.2.3 (Aug 25, 2021)
+## Fleet 4.2.4 (Sept 2, 2021)
+
+* Fix Redis Cluster support.
+
+* Handle properly the case where the agent opts is nil in the app config.
+
+* Improve how host users are stored to prevent deadlocks.
```

Comment:
```suggestion
* Improve how a host's `users` are stored in MySQL to prevent deadlocks. This information is available in the "Users" table on each host's **Host details** page and in the `GET /api/v1/fleet/hosts/{id}` API route.
```